File: vonage/php/create-video-session/create-video-session.php
<?php
require_once sprintf('%s/vendor/autoload.php', dirname(__DIR__, 1));

use OpenTok\{
    MediaMode,
    OpenTok,
};

$dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
$dotenv->load();

$api_key = $_ENV['VONAGE_API_KEY'];
$api_secret = $_ENV['VONAGE_API_SECRET'];

try {
    $opentok = new OpenTok($api_key, $api_secret);
    $options = [
        'mediaMode' => MediaMode::ROUTED,
    ];
    $session = $opentok->createSession($options);
    echo $session->getSessionId();
} catch (Exception $e) {
    echo sprintf('Exception %s: %s at line %d in %s',
        get_class($e), $e->getMessage(), $e->getLine(), $e->getFile()), "\n";
    exit(1);
}

exit;

File: vonage/python/create-video-session/create-video-session.py
import os
import sys
from os.path import dirname, join
from dotenv import load_dotenv
from vonage import Client

dotenv_path = join(dirname(__file__), '../../.env')
load_dotenv(dotenv_path)

application_id = os.environ.get('VONAGE_APPLICATION_ID')
application_private_key = os.environ.get('VONAGE_APPLICATION_PRIVATE_KEY')

try:
    client = Client(
        application_id=application_id,
        private_key=application_private_key,
    )
    session = client.video.create_session()
    print(session['session_id'])
except Exception as e:
    print('Exception %s: %s' % (type(e), e))
    sys.exit(1)

sys.exit
